hack/util.sh

```diff
@@ -183,8 +183,8 @@ function wait_for_url() {
 		fi
 		sleep $wait
 	done
-	echo "ERROR: gave up waiting for ${url}"
-	echo $(${cmd})
+	echo "ERROR: gave up waiting for ${url}" 1>&2
+	echo ${cmd} 1>&2
```

Comment:
Sorry, not what I meant.

When you do something like:
```sh
echo "$( ${commands} )"
```

You are:
 - spawning a subshell `$()`
 - running `${commands}` in that subshell
 - collecting stdout from it
 - asking `echo` to take that and put it into stdout

Instead, if you do simply:
```sh
${commands}
```

You then are:
 - running the `${commands}` and letting them touch `std{out,err}` as it would